### PR TITLE
Add SDK + UX Fixes for End-to-End DDR

### PR DESCRIPTION
**Pull request by tosmith97**
* Added cionic SDK + custom listeners for IMU and EMG predictions
* Game ends when the song ends
* Constant speed throughout the game (1 arrow every 2 seconds)
*Currently supports 1 song/difficulty
image

### CODE REVIEW

**Comment by dachelis** [comment]
BEATUFUL!

File: lib/bundle.js [hunk 1]
```diff
@@ -840,6 +833,81 @@ document.addEventListener("DOMContentLoaded", function () {
...
+    }
+  });
+
+  // connect to the gateway
+  document.getElementById('cionic-connect').onclick = function () {
+    let host = document.getElementById('host').value;
+    cionic.websocket(host);
+  };
+
+  // const monitorAPI = new cionicjs.MonitorAPI({});
```
Comment (ray-ray) [inline lib/bundle.js]: No monitor?

**Review by ray-ray — CHANGES_REQUESTED** [review]
Did you intentionally comment out the monitor code?

**Comment by tosmith97** [comment]
> Did you intentionally comment out the monitor code?

Yeah I usually comment out monitor code during local testing, otherwise it clogs the console with a bunch of 404s/403s

**Review by ray-ray — APPROVED** [review]

**Pull request merged**